utils/format

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+find examples -name "*.h"  -exec clang-format -i {} \;
```

Comment:
I believe clang-format can accept a list of files, which might be better, especially for those on Windows, where process creation can be slow. Perhaps not a big deal though.

Reply:
This seems like the easiest way to get the list of file names? I'd need to do something to loop over all dirs to find the files, yea?